components/ConsumedList/ConsumedList.js

```diff
@@ -8,23 +13,26 @@ export default function ConsumedList() {
   const minute = now.getMinutes();
 
   return (
-    <StyledList>
-      {dailyMeals.map((meal) => (
-        <StyledListItem key={meal.name}>
-          <span>{`${meal.name}`}</span>
-          <ShiftedSpan>{`${meal.calories} kcal`}</ShiftedSpan>
-          <span>{`${hour}:${minute}`}</span>
-          <button
-            style={{ border: "none", background: "none" }}
-            onClick={() => {
-              addDailyCount(-meal.calories);
-              deleteDailyMeal(meal.name);
-            }}
-          >
-            ❌
-          </button>
-        </StyledListItem>
-      ))}
-    </StyledList>
+    <CorrectionDiv>
+      <StyledList>
+        {dailyMeals.map((meal) => (
+          <StyledListItem key={meal.name}>
+            <span>{`${meal.name}`}</span>
+            <ShiftedSpan>{`${meal.calories} kcal`}</ShiftedSpan>
+            <span>{`${hour}:${minute}`}</span>
+            <button
+              style={{ border: "none", background: "none" }}
+              onClick={(event) => {
+                event.stopPropagation();
```

Comment:
why are we stopping propagation here?

Reply:
So that I can delete Items without closing the opened list